openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/routing/ResultTest.java

```diff
@@ -3651,6 +3653,177 @@ public void expectMaxpeedHgvForward() {
+        Response res = given()
+            .header("Accept", "application/json")
+            .header("Content-Type", "application/json")
+            .pathParam("profile", getParameter("ptProfile"))
+            .body(body.toString())
+            .when()
+            .post(getEndPointPath() + "/{profile}")
+            .then().log().ifValidationFails()
+            .assertThat()
+            .body("any { it.key == 'routes' }", is(true))
+            .body("routes.size()", is(3))
+            .body("routes[0].summary.transfers", is(2))
+            .body("routes[0].legs.size()", is(5))
+            .body("routes[0].legs[0].containsKey('arrival')", is(true))
+            .body("routes[0].legs[0].containsKey('departure')", is(true))
+            .body("routes[0].legs[0].containsKey('instructions')", is(true))
+            .body("routes[0].legs[0].containsKey('geometry')", is(true))
+            .body("routes[0].legs[1].containsKey('feed_id')", is(true))
+            .body("routes[0].legs[1].containsKey('trip_id')", is(true))
+            .body("routes[0].legs[1].containsKey('route_id')", is(true))
+            .body("routes[0].legs[1].containsKey('route_type')", is(true))
+            .body("routes[0].legs[1].containsKey('route_desc')", is(true))
+            .body("routes[0].legs[1].containsKey('geometry')", is(true))
+            .statusCode(200).extract().response();
+
+        given()
+            .header("Accept", "application/geo+json")
+            .header("Content-Type", "application/json")
+            .pathParam("profile", getParameter("ptProfile"))
+            .body(body.toString())
+            .when()
+            .post(getEndPointPath() + "/{profile}/geojson")
+            .then().log().ifValidationFails()
+            .assertThat()
+            .body("any { it.key == 'features' }", is(true))
+            .body("features.size()", is(3))
+            .body("features[0].properties.transfers", is(2))
+            .body("features[0].properties.legs.size()", is(5))
+            .body("features[0].properties.legs[0].containsKey('arrival')", is(true))
+            .body("features[0].properties.legs[0].containsKey('departure')", is(true))
+            .body("features[0].properties.legs[0].containsKey('instructions')", is(true))
+            .body("features[0].properties.legs[0].containsKey('geometry')", is(true))
+            .body("features[0].properties.legs[1].containsKey('feed_id')", is(true))
+            .body("features[0].properties.legs[1].containsKey('trip_id')", is(true))
+            .body("features[0].properties.legs[1].containsKey('route_id')", is(true))
+            .body("features[0].properties.legs[1].containsKey('route_type')", is(true))
+            .body("features[0].properties.legs[1].containsKey('route_desc')", is(true))
+            .body("features[0].properties.legs[1].containsKey('geometry')", is(true))
+            .statusCode(200).extract().response();
+    }
+
+    @Test
+    public void testPTFail() {
```

Comment:
Either split in multiple tests or parametrize the tests.

Reply:
Parametrized.